src/components/event/Overview.vue

```diff
@@ -73,7 +79,9 @@ export default Vue.extend({
      * @returns {string | undefined}
      */
     lang(): string | undefined {
-      return this.event.catcherType ? this.event.catcherType.split('/').pop() : '';
+      return this.event.catcherType
+        ? this.event.catcherType.split("/").pop()
+        : "";
```

Comment:
We decided to remove `catcherType`, you can remove this code.

Reply:
> We decided to remove `catcherType`, you can remove this code.

@GeekaN2 Told me to remove it since we removed `catcherType`. @neSpecc